mojo/core/node_controller.cc

```diff
@@ -365,6 +365,18 @@ bool NodeController::SyncSharedBuffer(
   return true;
 }
 
+bool NodeController::SyncSharedBuffer2d(const base::UnguessableToken& guid,
+                                        size_t offset,
+                                        size_t sync_size,
+                                        size_t width,
+                                        size_t stride) {
+  if (broker_)
+    return broker_->SyncSharedBuffer2d(guid, offset, sync_size, width, stride);
+
+  // Normal path with Unix domain socket on browser(host)
```

Comment:
"broker_ is null" means that this is the browser process. The browser process is not likely to send tile(2d) data. I think we need NOTREACHED() here.

Reply:
CHECK(broker_) also looks good.

Reply:
Done. Thanks.